.github/workflows/tests.yml

```diff
@@ -36,6 +36,22 @@ jobs:
         with:
           filters: .github/package-filters/rs-packages.yml
 
+  build-rust-deps:
```

Comment:
I just realised that it will be more efficient to build this lib in grovedb release only once. We can do that in the next PR

Reply:
added comment